src/main/java/net/juniper/netconf/Device.java

```diff
@@ -12,11 +12,13 @@
 import com.jcraft.jsch.JSch;
 import com.jcraft.jsch.JSchException;
 import com.jcraft.jsch.Session;
+import java.util.Arrays;
```

Comment:
nit (you don't need to change)

I would have added this down around line 33, right above the import for java.util.List.

Reply:
Will do.